examples/provider-install-verification/main.tf

```diff
@@ -0,0 +1,16 @@
+
+terraform {
+  required_providers {
+    mongodb = {
+      source = "hashicorp.com/edu/mongodb-driver"
```

Comment:
nit: It'd be nice to have a more personalized URL

Reply:
Ah yes I should update this one. This naming is from the tutorial I followed.

The name should be something like `registry.terraform.io/riskident/mongodb-driver`, depending on the username we can register at registry.terraform.io.

Will change